.github/ISSUE_TEMPLATE/missing-test-case-en.yaml

```diff
@@ -0,0 +1,111 @@
+---
+name: Missing Test Case
+description: Report a missing test case to help us improve our content.
+title: "Missing Test Case - <problem ID and name>"
+labels: ["LCUS", "watchlist", "problem"]
+assignees:
+  - LC-Pam
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this report!
+  - type: input
+    id: username
+    attributes:
+      label: LeetCode username
+      description: >
+        Please enter your LeetCode username so that we can reward
+        your contribution when the issue is resolved.
+    validations:
+      required: false
```

Comment:
Users may want to contribute anonymously, so a username is not required.

Reply:
hello
how are you

Reply:
I'm doing just about fine, what about you @mirodil06 ? May I ask what is your connection with LeetCode?

Reply:
bu haqida so'ramen!
bilishingiz shartmas

Reply:
qaysi loyha ustida ishlavos

Reply:
hozirgi kunda

Reply:
siz botmisiz yoki odam
bunaqa savol o'rinsiz lekin bilishim kere